cgmes/cgmes-model/src/main/resources/CIM16.sparql

```diff
@@ -121,7 +121,7 @@ SELECT *
 
 # query: substations
 SELECT ?Substation ?name ?SubRegion ?subRegionName ?Region ?regionName
-{ GRAPH ?graph {
+{
```

Comment:
I made the test with 5 IGM + the boundary set I have, no noticeable performance difference between the to options (always around 30-35 ms)

Note that cim:SubgeographicalRegion-s may also exist in boundary set, so functionnaly speaking my call would be not to restrict Substations and SubGeographicalRegions to be in the same graph.

See also [QoCDC](https://eepublicdownloads.azureedge.net/clean-documents/digital/QualityOfCGMESdatasetsAndCalculations_v3_3.pdf) line 641 with the example of Denmark. Storebælt HVDC link between DKW and DKE will have to be in a SubGeographicalRegion. Since this HVDC SubGeographicalRegion will have to be referenced by both DKW and DKE IGMs, it is more than likely that this SubGeographicalRegion will be placed in boundary set.

Reply:
I prefer using `GRAPH` as it make it easier from a user viewpoint to understand in which file/profile one can find the objects and their attributes (for example, to understand that the reference to the sub-region is in the same file as where the substation is defined even if the sub-region itself is elsewhere) which is interesting for debug purposes.

For the second use case you propose, I think we can do this:
```sparql
# query: substations
SELECT ?Substation ?name ?SubRegion ?subRegionName ?Region ?regionName
{ GRAPH ?graph {
    ?Substation
        a cim:Substation ;
        cim:IdentifiedObject.name ?name ;
        cim:Substation.Region ?SubRegion .
}
GRAPH ?graph2 {
    ?SubRegion
        cim:SubGeographicalRegion.Region ?Region ;
        cim:IdentifiedObject.name ?subRegionName .
}
GRAPH ?graph3 {
    ?Region cim:IdentifiedObject.name ?regionName .
}
}
```

Reply:
sounds good, I updated this way, thanks!